README.md

```diff
@@ -1,2 +1,25 @@
-# advisory-group
-Enterprise Advisory Group
+# Enterprise Advisory Group
+
+The Enterprise Advisory Group consists of corporate representatives in leadership roles responsible for managing for large-scale deployments of Node.js in an Enterprise environment. We strive to provide timely feedback to the [TSC](https://github.com/nodejs/TSC), the [CommComm](https://github.com/nodejs/community-committee), the [Node.js project](https://github.com/nodejs/node) and the Node.js Foundation about the needs of Node.js Enterprise users.
+
+Thank you for your interest and support. Please [create an issue](https://github.com/nodejs/user-feedback/issues) in this repo if you are interested in dedicating 5+ hours a month during the formation process.
+
+### Governance and Contributing
+The Enterprise Advisory Group has adopted the core governance and contributing policies of Node.js. This group is a part of the [Node.js End User Feedback Initiative](https://github.com/nodejs/user-feedback) which is under the charter of the [Node.js Community Committee (CommComm)](https://github.com/nodejs/community-committee).
+
+Members are expected to provide feedback on how proposed changes in the Node.js project will affect Enterprise teams, business goals, and application performance at scale. Members attend regular group meetings to provide feedback and respond to occasional outreach requests. 
```

Comment:
How is the TSC expected to ask for feedback? How is the agenda decided? What is the result of the feedback and what kind of expectations should the group have for results?

Reply:
Again, none of this needs to be documented in the charter and is rightfully determined by the group that actually forms.

Reply:
Perhaps a small blurb with expectation setting could be helpful?

`While the TSC does not guarantee particular action based on feedback they do intend to use it to make informed decisions`

/cc @nodejs/tsc

Reply:
Again, I'd like to note that @dshaw commented that this is not going to be a WG (ref: https://github.com/nodejs/enterprise-advisory-group/pull/5#issuecomment-352534359). If it _were_ a WG, it would not fall under the User Feedback Initiative, but be it's own independent group (in the CommComm an initiative is less empowered than a WG).